Makefile

```diff
@@ -364,6 +364,7 @@ android-test:
 component-test-watch: export TARGET := clojure
 component-test-watch: export COMPONENT_TEST := true
 component-test-watch: export BABEL_ENV := test
+component-test-watch: export JEST_USE_SILENT_REPORTER := false
```

Comment:
would be nice if we can call these with a shorthand arg, e.g 
`make component-test-watch --silent` or something to that effect.

Reply:
Good idea @J-Son89, but to achieve this DX is impossible I think with `make`. The argument must be passed with a variable, like `make component-test-watch ARG1=--silent`. Maybe we should start to consider scripts instead to create a better CLI, we could even write some with babashka, something to consider cc @yqrashawn